### PR TITLE
Bump SDK to 0.7.0

### PR DESCRIPTION
**Pull request by maili-cb**
Fixes # .

### Motivation
Bump SDK to 0.7.0
### Solution
Bump SDK to 0.7.0

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -158,6 +158,7 @@ func main() {
 		[]*types.NetworkIdentifier{cfg.Network},
 		nil,
 		services.MempoolCoins,
+		``,
```
Comment (shrimalmadhur) [inline main.go:161]: Is this similar to empty string?

**Comment by shrimalmadhur** [comment]
Did you fork this branch out of #80 ? - It looks like it has all it's changes?

**Comment by maili-cb** [comment]
> Did you fork this branch out of #80 ? - It looks like it has all it's changes?

Keeping all the changes because the cmd are changed in the Makefile and new upgrades will have dependencies on the cmd change, need to merge #80 first and then merge this one

File: main.go (hunk 1 above)
Comment (maili-cb) [reply main.go]: yes, but it seems backtick in Go stands for raw string. Changed to double quote now

**Review by shrimalmadhur — APPROVED** [review]

**Pull request merged**